docs/software/ros.mdx

```diff
@@ -1,9 +1,26 @@
 ---
-title: ROS
-sidebar_position: 1
+title: Robot Operating System (ROS)
+sidebar_label: ROS
+sidebar_position: 2
 ---
 
-TODO
+Clearpath Robotics is committed to the [Robot Operating System (ROS)](https://www.ros.org/) and the community of researchers
+and users that support it. Each robot that Clearpath deploys is designed to be used with this open
+source platform. The extensibility of ROS has also allowed Clearpath to use it to develop the proprietary
```

Comment:
remove 'the':
'...allowed Clearpath to use it to develop proprietary autonomy software...'

Reply:
Updated